DEV_GUIDE.md

```diff
@@ -318,3 +319,14 @@ mvn org.asciidoctor:asciidoctor-maven-plugin:process-asciidoc@convert-to-html
 
 The output will be in `target/html/master.html`. 
 
+## Using the GitHub CI workflows against a fork
+
+All CI [workflows](.github/workflows) defined within  will execute against your fork, apart from [docker workflow](.github/workflows/docker.yml).
+To enable the docker worklflow, you need to configure two repository [variables](https://docs.github.com/en/actions/learn-github-actions/variables)
+and one respository [secret](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).
+
+* `QUAY_ORG` variable - your Quay organisation.
+* `QUAY_USER` variable - your Quay username (or robot account).
+* `QUAY_TOKEN` secret - the token that corresponds to `QUAY_USER`.
+
+The workflow will push to `${QUAY_ORG}/kroxylicious-developer` so ensure that the Quay user has sufficient privileges. 
```

Comment:
Not strictly part of this PR but should we also include some hints about registering for a free Quay account? We probably can't get away with assuming everyone already has one...

Reply:
I can add that near the prerequisites section. 

Probably best not to assume Quay.io. User could use docker.io or ghcr or something else.

With that in mind, I think to rejig the variables/secret to use container registry neutral language

* `QUAY_USER` -> `REGISTRY_USERNAME` 
* `QUAY_TOKEN` -> `REGISTRY_TOKEN`
* `QUAY_ORG` -> `REGISTRY_DESTINATION` (quay.io/kroxylicious/kroxylicious-developer) (
* REGISTRY_SERVER eg. docker.io or quay.io

justification for names:

1. both `docker login` and `podman login` CLIs use the term `username` rather than `user`
2. podman push CLI  term uses term `destination`   (docker push uses `name` but that too general)
3. podman login uses term `server` docker login uses the term `registry`.  

How's that sound?

Reply:
Yeah that sounds very reasonable

Reply:
@SamBarker I made that change.   could you re-review please  I'd like to refactor the container registry interaction in `/scripts/run-with-strimzi.sh`  etc to follow the same pattern, but this is better in a separate PR.